lazycatlabs/flutter_auth_app
# [REFACTOR] Feature First

## What it does

- Refactor code from **layer first** to **feature first**

## How to test

1. Clone the project
2. Run the project
3. Observe project structure

## Screenshot

![image](https://user-images.githubusercontent.com/1531684/205448792-f3952257-07f2-4152-b95a-99e115096079.png)
